### ISSUE TITLE
Debian Key expired at Suse Build Service

### ISSUE THREAD

**Issue by sysms** [lutris/lutris#4982]
### Bug description

Dear maintainer(s),

I just want to inform you, that the key files at

https://download.opensuse.org/repositories/home:/strycore/Debian_10/Release.key
https://download.opensuse.org/repositories/home:/strycore/Debian_11/Release.key
https://download.opensuse.org/repositories/home:/strycore/Debian_12/Release.key

have expired again (see https://github.com/lutris/lutris/issues/3562).

Please renew them. :) 

Thanks!

### How to Reproduce

1. Follow the instructions at https://software.opensuse.org/download.html?project=home%3Astrycore&package=lutris

### Expected behavior

Install the package without problems.

### Log output

```shell
n/a
```

### System Information

```shell
n/a
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by strycore** [comment]
I ran `make pgp-renew` which does 

```
	osc signkey --extend home:strycore
	osc rebuildpac home:strycore --all
```

Is everything ok now?

**Comment by ACR-Jeff** [comment]
I had the same issue yesterday, I went to the link and copied the repository and ran it again, It worked for me and updated, [](https://software.opensuse.org/download.html?project=home%3Astrycore&package=lutris)

**Comment by Conusoid** [comment]
Please update the instructions on [this page](https://lutris.net/downloads). When my key was located in /etc/apt/keyrings/lutris.gpg, I only got these update errors:

```
W: GPG error: http://download.opensuse.org/repositories/home:/strycore/Debian_12 InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2F7F0DA5FD5B64B9
E: The repository 'http://download.opensuse.org/repositories/home:/strycore/Debian_12 InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Apparently, the name of the key file and/or its location matter in this case. Everything worked as soon as I changed them to the ones listed [here](https://software.opensuse.org/download.html?project=home%3Astrycore&package=lutris).

**Comment by Maxwellfire** [comment]
@Conusoid the correct location depends on the `signed-by` parameter in your sources list. If you have `/etc/apt/keyrings/lutris.gpg` listed there (because you originally used the lutris instructions instead of the Debian instructions), then placing the updated file at that location is correct.
